.github/workflows/pr-build-live-branch.yml

```diff
@@ -0,0 +1,132 @@
+          sed -i "s/^Stable tag: .*$/Stable tag: $DEV_VERSION/" readme.txt
+
+      - name: "Build the plugin"
+        id: build_plugin
+        uses: ./.github/actions/build
+
+      - name: "Update the structure of artifacts for Jetpack Beta Builder" 
+        run: |
+          cd release
+          mv woocommerce-payments woocommerce-payments-dev
+          zip -q -r "woocommerce-payments-dev.zip" "woocommerce-payments-dev/"
+          rm -fR "woocommerce-payments-dev"
+
+      - name: "Add file size notice"
+        run: |
+          echo ":information_source: Ignore the artifact size mentioned since GitHub calculates the size of the source folder instead of the zip file created." >> $GITHUB_STEP_SUMMARY
+
+      - name: "Upload the zip file as an artifact"
+        uses: actions/upload-artifact@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: plugins # Hard-coded value for Jetpack Beta Builder.
+          path: release
+          # Only need to retain for a day since the Jetpack Beta Builder slurps it up to distribute.
+          retention-days: 1
+
+      - name: "Get plugin data for Jetpack Beta Builder"
+        id: get-plugin-data
+        env:
+          DEV_VERSION: ${{ steps.get-dev-version.outputs.DEV_VERSION }}          
+        run: |
+          # Plugin data is passed as a JSON object.
+          PLUGIN_DATA="{}"          
+          PLUGIN_DATA=$( jq -c --arg slug "woocommerce-payments" --arg ver "$DEV_VERSION" '.[ $slug ] = { version: $ver }' <<<"$PLUGIN_DATA" )
+          echo "plugin-data=$PLUGIN_DATA" >> $GITHUB_OUTPUT
+
+      - name: "Inform Jetpack Beta Builder webhook"
+        if: steps.get-plugin-data.outputs.plugin-data != '{}'
+        env:
+          SECRET: ${{ secrets.WCPAYBETA_SECRET }}
+          PLUGIN_DATA: ${{ steps.get-plugin-data.outputs.plugin-data }}
+          PR: ${{ github.event.number }}
+        run: |
+          curl -v --fail -L \
+            --url "https://betadownload.jetpack.me/gh-action.php?run_id=$GITHUB_RUN_ID&pr=$PR&commit=$GITHUB_SHA" \
+            --form-string "repo=$GITHUB_REPOSITORY" \
+            --form-string "branch=${GITHUB_REF#refs/heads/}" \
+            --form-string "plugins=$PLUGIN_DATA" \
+            --form-string "secret=$SECRET"
+
+      - name: "Add/update the information as a comment in the PR"
+        env:
+          GITHUB_TOKEN: ${{ secrets.BOTWOO_TOKEN }}
+          PR_NUMBER: ${{ github.event.number }}
+          PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+          PR_HEAD_REF: ${{ github.head_ref }}
+        run: |
+          BODY="### Test the build on Jurassic Ninja 
+          
+          This feature is only available for A12s. Please log in into https://jurassic.ninja/specialops/ before clicking the link below:
+          
+          https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=${PR_HEAD_REF}
```

Comment:
Possible to make this a `href` with let's say:
```
[Launch JN with branch](link)
```

Reply:
Addressed 593486aad251d44f87f4245d11693960396e1bf8